core/station/impl/src/mappers/authorization.rs

```diff
@@ -380,6 +382,32 @@ impl From<&station_api::CreateRequestInput> for Resource {
     }
 }
 
+impl From<&CanisterSnapshotsInput> for Resource {
+    fn from(input: &CanisterSnapshotsInput) -> Self {
+        let canister_id = input.canister_id;
+        if let Ok(()) = EnsureExternalCanister::is_external_canister(canister_id) {
```

Comment:
using .is_ok() instead would be cleaner IMHO

Reply:
I went for the latter in this [commit](https://github.com/dfinity/orbit/pull/553/commits/3b00ebe020f57e519cb2ecde600e3114f25f6569) since the main motivation for `let Ok(())` was to prevent accidental changes to the API, e.g., making it return `Result<bool, ...>` and forgetting to consider the inner Boolean value at this call site.